packages/pigeon/pigeons/android_unittests.dart

```diff
@@ -1,30 +0,0 @@
-// Copyright 2013 The Flutter Authors. All rights reserved.
```

Comment:
I forgot to delete this file in my last consolidation; it's unused. I was getting warnings in my stale output copy and couldn't figure out why at first since the source file was still there.

(Also: you'll need to clean your generated files again before this lands if you haven't done so recently.)